.buildkite/cicd.rayci.yml

```diff
@@ -5,3 +5,9 @@ steps:
       # TODO(aslonnie): wrap this in a script, and upload test telemetry.
       - bazel test --test_tag_filters=ci_unit //ci/ray_ci/...
     depends_on: forge
+
+  - label: "CI/CD: release test infra"
+    commands:
+      # TODO(aslonnie): wrap this in a script, and upload test telemetry.
+      - bazel test --test_tag_filters=release_unit //release/...
```

Comment:
can we still run the ./ci/build/upload_build_info.sh here? or this pipeline doesn't have permission?

Reply:
I think the lambda will need an update I think. I am also not sure if `upload_build_info.sh` has all its deps installed.

Reply:
Got you, good enough for these tests!